etc/generic/requirements-elyra.txt

```diff
@@ -17,6 +17,7 @@ requests==2.31.0
 tornado==6.3.3
 traitlets==5.1.1
 urllib3==1.26.17
+pendulum==2.1.2
```

Comment:
this is probably wrong: pendulum is not required for Elyra itself, but when running the rendered DAG definition on Airflow. And Airflow as it seems already has the library in its worker and scheduler containers. Or at least my runtime containers have it. Need to research a little about this.

Reply:
@romeokienzler ready for review